Doc/library/optparse.rst

```diff
@@ -952,6 +952,8 @@ The canonical way to create an :class:`Option` instance is with the
    you may also supply :attr:`~Option.type` and :attr:`~Option.dest` option
    attributes; see :ref:`optparse-standard-option-actions`.)
 
+.. class:: Values
+
 As you can see, most actions involve storing or updating a value somewhere.
 :mod:`optparse` always creates a special object for this, conventionally called
 ``options`` (it happens to be an instance of :class:`optparse.Values`).  Option
```

Comment:
IMO, it looks a little odd and potentially confusing to just have the option class definition sitting standalone like this without any explicit description, leaving readers to try to infer it from the context of the following paragraphs. While it would be ideal to rewrite this section to be a proper API reference doc per the Diataxis guidelines, which would avoid this issue and make this fit in much better, that's way out of scope here.

Instead, I suggest we either add a short description of what the `Values` class actually is in the body of this directive, e.g.

```suggestion
.. class:: Values

   An object holding parsed argument names and values as attributes.
   Normally created while calling when calling :meth:`OptionParser.parse_args` rather than directly,
   and can be overridden by a custom subclass passed to the *values* via the *option_class* argument
   to :meth:`OptionParser.parse_args` (as described in :ref:`optparse-parsing-arguments`).
```
  
Or, alternatively, we could just not add this now and leave these warnings to be addressed in a dedicated followup:

```suggation
```

Reply:
First one looks good, thanks.